benchmarks/mmmu/mmmu.py

```diff
@@ -0,0 +1,169 @@
+import ast
+import base64
+from io import BytesIO
+
+from datasets import concatenate_datasets, load_dataset
+from PIL import Image
+
+from inspect_ai import Task, task
+from inspect_ai.dataset import Sample
+from inspect_ai.dataset._dataset import MemoryDataset
+from inspect_ai.model import ChatMessageUser, ContentImage, ContentText, GenerateConfig
+from inspect_ai.scorer import choice, model_graded_fact
+from inspect_ai.solver import generate, multiple_choice, prompt_template
+
+
+# Helper function to convert images to data URLs
+def image_to_data_url(image):
+    if isinstance(image, Image.Image):
+        buffered = BytesIO()
+        image.save(buffered, format="PNG")
+        image_bytes = buffered.getvalue()
+    else:
+        with open(image, "rb") as image_file:
+            image_bytes = image_file.read()
+
+    encoded_string = base64.b64encode(image_bytes).decode("utf-8")
+    mime_type = "image/png"
+    return f"data:{mime_type};base64,{encoded_string}"
+
+
+# Helper function to create content list from record
+def create_content_list(record):
```

Comment:
You might want to move this comment into a [docstring](https://peps.python.org/pep-0257/#one-line-docstrings).

Reply:
Done in the factored-out function for this.